### PR TITLE
Ipxe lz

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

File: blog/content/post/2020-05-14-ipxe_lz_support.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+title: Starting TrenchBoot's Landing Zone from iPXE
+abstract: 'In this article we present support for starting Landing Zone from
+          another bootloader: iPXE. It may not be as featureful as GRUB2, but
+          it has enough juice to start DRTM using images obtained from a remote
+          server'
+cover: /covers/trenchboot-logo.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2020-05-14
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-ipxe_lz_support.md:11]: Do we want to change the date?

File: blog/content/post/2020-05-14-ipxe_lz_support.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+Code can be found in our iPXE fork, [landing_zone branch](https://github.com/3mdeb/ipxe/tree/landing_zone).
+It can be build either as a standalone binary or as a coreboot PXE ROM. Let's
+start with the latter one.
+
+#### Embedded in coreboot
+
+Follow [HTTPS enabling tutorial](https://blog.3mdeb.com/2020/2020-05-06-ipxe-https/),
+with two important changes:
+
+* clone from PC Engines repository and switch to proper branch (TODO: tag):
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-ipxe_lz_support.md:59]: I guess we don't want the TODO tag? @krystian-hebel

File: blog/content/post/2020-05-14-ipxe_lz_support.md [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+
+```bash
+git clone --recurse-submodules https://github.com/pcengines/coreboot.git
+cd coreboot
+checkout ???
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-ipxe_lz_support.md:64]: Same as above with ??? @krystian-hebel

File: blog/content/post/2020-05-14-ipxe_lz_support.md (hunk 1 above)
Comment (krystian-hebel) [reply blog/content/post/2020-05-14-ipxe_lz_support.md]: Changed to today's date

File: blog/content/post/2020-05-14-ipxe_lz_support.md (hunk 2 above)
Comment (krystian-hebel) [reply blog/content/post/2020-05-14-ipxe_lz_support.md]: Done

File: blog/content/post/2020-05-14-ipxe_lz_support.md (hunk 3 above)
Comment (krystian-hebel) [reply blog/content/post/2020-05-14-ipxe_lz_support.md]: Done and written as one command

File: blog/content/post/2020-05-14-ipxe_lz_support.md (hunk 1 above)
Comment (artur-rs) [reply blog/content/post/2020-05-14-ipxe_lz_support.md]: I always update the date and filename-date before the publishing

**Pull request merged**